docs/.vuepress/config.js

```diff
@@ -7,14 +7,16 @@ module.exports = {
     // ],
     sidebar: [
       "/",
-      "/markup-language/"
+      "/markup-language/",
+      "/api-client/",
+      "/openlaw-object/"
       // add new top level sections here...
     ],
     algolia: {
       apiKey: "68c3c0d36676a3acce1cd3c7087bc7c9",
```

Comment:
@mroth just wanted to bump this as we're getting close to making the protocol docs go live.

Reply:
Nope. It’s basically just an ID, it’s fine / intended to be publicly readable since it’s on the frontend to query DocSearch.
(Thanks for the ping. Somehow I missed this until now.)

Reply:
sounds good. thanks for clarifying

Reply:
thanks! glad to hear it